test/xpu/nn/test_safe_softmax.py

```diff
@@ -0,0 +1,44 @@
+import torch
+from torch.testing._internal.common_utils import TestCase
+
+cpu_device = torch.device("cpu")
+xpu_device = torch.device("xpu")
+
+
+class TestSafeSoftMax(TestCase):
```

Comment:
If it is not present in the stock PyTorch tests, please move it to `test/regression/`

Reply:
done